packages/plaid-funding-source/pages/index.tsx

```diff
@@ -120,7 +145,7 @@ const Home: NextPage = () => {
         token: linkToken
     });
 
-    const isReady = customerId && fundingSourceName && isPlaidLinkReady;
+    const isReady = customerId && fundingSourceName && isChecked && isPlaidLinkReady;
```

Comment:
Though this does work in its current state, might be worth putting an additional guard around ensuring that `isChecked` and `odaLink` are valid since just `isChecked` could be true even when `odaLink` is null, which would cause the POST request to throw NPE.